### PR TITLE
WL-3887 Adding macro for standard flash notification message

### PR DESCRIPTION
**Pull request by surya-janani**
setting message through vmReference and

### CODE REVIEW

File: tool/src/java/org/sakaiproject/cheftool/VelocityPortletPaneledAction.java [hunk 1]
```diff
@@ -1095,7 +1133,7 @@ protected void setVmStdRef(HttpServletRequest request, HttpServletResponse respo
 			if (session.getAttribute(ATTR_TOP_REFRESH) != null)
 			{
 				setVmReference("topRefresh", Boolean.TRUE, request);
-				session.removeAttribute(ATTR_TOP_REFRESH);
```
Comment (buckett) [inline tool/src/java/org/sakaiproject/cheftool/VelocityPortletPaneledAction.java:1098]: Why are you removing the removal of the attribute.

**Comment by buckett** [comment]
I don't see anything to deal with the top frame refresh, should I?

**Comment by surya-janani** [comment]
No .Top refresh is in SiteAction. https://github.com/ox-it/wl-site-manage/pull/47

File: tool/src/java/org/sakaiproject/cheftool/VelocityPortletPaneledAction.java (hunk 1 above)
Comment (buckett) [inline tool/src/java/org/sakaiproject/cheftool/VelocityPortletPaneledAction.java:1098]: Doesn't this mean that other tools wanting a top refresh will stop working?

File: tool/src/java/org/sakaiproject/cheftool/VelocityPortletPaneledAction.java [hunk 2]
```diff
@@ -470,6 +493,19 @@ protected void toolModeDispatch(String methodBase, String methodExt, HttpServlet
...
+				String sNotif = (String) getState(req).getAttribute(STATE_NOTIF);
+				if (sNotif != null)
+				{
+					sbNotif.append(sNotif);
+					getState(req).removeAttribute(STATE_NOTIF);
+				}
+				if (sbNotif.length() > 0)
+				{
+					setVmReference("flashNotif", sbNotif.toString(), req);
+					setVmReference("flashNotif.close.title","Click to dismiss",req);
```
Comment (buckett) [inline tool/src/java/org/sakaiproject/cheftool/VelocityPortletPaneledAction.java]: This needs i18n from a resource bundle.

**Comment by surya-janani** [comment]
Making review changes and rebased

**Pull request merged**